examples/create_account.rs

```diff
@@ -186,70 +187,39 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     println!("    Public Key: {}", new_key_pair.public_key());
     println!("       Deposit: {}", initial_deposit);
     println!("-------------------------------------------------------------");
+    println!("       Tx Hash: {}", transaction.get_hash_and_size().0);
+    println!("=============================================================");
 
-    let request = methods::broadcast_tx_async::RpcBroadcastTxAsyncRequest {
+    let request = methods::send_tx::RpcSendTransactionRequest {
         signed_transaction: transaction.sign(&signer),
+        wait_until: TxExecutionStatus::Final,
```

Comment:
@telezhnaya Will it wait indefinitely? There used to be a standard timeout after 10 seconds. It might be bad for nearcore node to handle such requests without timeouts. It is fine for the client code to retry

Reply:
Valid concern. Reverted